docs/Actions/grant-user-role.md

```diff
@@ -0,0 +1,248 @@
+---
+id: grant-user-role
+title: Grant User Role
+sidebar_label: Grant User Role
+---
+
+> Audience: [`Business Users`](/docs/audience#business-users)<br/>
+> Skill Prerequisites: `Permission Scheme`
+
+## `Typical Use Cases`
+
+- Assign roles to new users on registration
+- Give access to resources for specific users
+- Temporary give access to users
+
+## `Related Actions`
+
+| Action Name                                             | Description                                                        |
+| ------------------------------------------------------- | ------------------------------------------------------------------ |
+| [Load Users](/docs/Actions/load-user.md)                | Can be used to grant roles to a single user or to a list of users. |
+| [User Registration](/docs/Actions/user-registration.md) | Can be used to grant roles to a newly created user.                |
+
+## `Input Parameter Reference`
+
+| Parameter        | Description                                                                                                                                                                                                                                                  | Supports Tokens | Default      | Required |
+| ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | --------------- | ------------ | -------- |
+| Role             | Select the security role to grant from a list with all the roles in the application. RoleId can also be used as input.                                                                                                                                       | Yes             | Not selected | Yes      |
+| Other Role Names | Freely input the role name(s) you want to grant. Multiple roles are separated by comma. RoleId can also be used as input.                                                                                                                                    | Yes             | Empty string | No       |
+| Role Validity    | Set a validity period to the granted role. The validity period can be done: for a number of days, date interval with or without tokens. <br></br> When the validity period will expire, the user will not have the permissions of that role anymore. | No              | Not selected | No       |
```

Comment:
There also is the extend from current validity, offset from now. You should go a bit more into detail about the modes a role validity can be input.